integration_tests/dbt_project.yml

```diff
@@ -83,4 +107,55 @@ seeds:
       +column_types:
         id: "{%- if target.type == 'bigquery' -%} INT64 {%- else -%} bigint {%- endif -%}"
         order_id: "{%- if target.type == 'bigquery' -%} INT64 {%- else -%} bigint {%- endif -%}"
-        user_id: "{%- if target.type == 'bigquery' -%} INT64 {%- else -%} bigint {%- endif -%}"
\ No newline at end of file
+        user_id: "{%- if target.type == 'bigquery' -%} INT64 {%- else -%} bigint {%- endif -%}"
+    shopify_abandoned_checkout_discount_code_data:
```

Comment:
what were the seed files needed for in this PR? is this data being brought up into the `orders` and `customers` models and needed for testing across DBs?

Reply:
the seed files were just for the new tables. because this PR is pointing to my working branch of the source package, Buildkite expected those new tables when running the upstream source package stuff

Reply:
so the seeds aren't super relevant, this just came up because this is my first PR on the transform package for this project

Reply:
ah got it, thanks for clarifying!